lib/views/auth/login_view.dart

```diff
@@ -137,6 +140,7 @@ class LoginScreenState extends State<LoginScreen> {
                     InkWell(
                       onTap: () async {
                         debugPrint('Login!');
+                        emailValidation();
                         errorMessage = null;
                         if (_formKey.currentState.validate()) {
```

Comment:
On this line we validate the form and we see the textformfileds shrinked with an error message, Instead of that show error message using snackbar or simple text below the form. That shrink doesn't look good.

Reply:
> On this line we validate the form and we see the textformfileds shrinked with an error message, Instead of that show error message using snackbar or simple text below the form. That shrink doesn't look good.

@himanshusharma89 Are you telling about for password validation?

Reply:
@Neha62-lit No, I am talking about the validation which compresses the height of textfields when it is in-valid. Instead of that error on each textformfield, you can show the error message below the social auth icons.

Reply:
Okay